### DISCUSSION TITLE
Why can't they put our outfits visible to other players

### DISCUSSION THREAD

**Discussion by Sbst397** [thatskymod/Sky-CotL-Scripts#126] [Q&A]
I was wondering why can't other players see our outfits why is it invisible and I wonder if someone is working on this because i think we're a lot wanting that our outfits become visible to other players, could you do something about that?

**Comment by thatskymod** [comment]
Hello,

as @ObservantLurker correctly mentioned, even though mods can theoretically unlock outfits on your end, this has no effect on how others see you. This is because outfits you don't own are immediately discarded as soon as they reach the remote server. Even with best efforts, there isn’t anything mod makers can do. Programming their own private server was once considered a potential solution but never completed due to its vast complexity.

**Reply by Sbst397** [reply]
Thanks for the informations

**Comment by Saatoooo** [comment]
Alguien sabe por qué no se completan las diarias hon el mod?

**Reply by thatskymod** [reply]
Hello, and thank you for the notice.

I recommend checking out the newly released **Dukun Langit v0.4.2**, which includes several updates and improvements:

- Fixed Daily/Seasonal Quests
- Resolved minor bugs
- New feature: "F Red Dots" (Clear all red dots for emotes and poses)

**Supported Sky Versions:**
- ✅ Sky Live 0.26.5 (280889)
- ❌ Sky Beta (Not updated)

You can download the update from [this link](https://github.com/thatskymod/Sky-CotL-Scripts/blob/b0ded79533ae3135767feb03afe9c579d0d20689/Canvas/Libs/libLangit042.so).

**Comment by thatskymod** [comment]
As the question has already been answered in https://github.com/thatskymod/Sky-CotL-Scripts/discussions/126#discussioncomment-10520675 and documented in KNOWLEDGE.md: [Outfit Visibility](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/KNOWLEDGE.md#outfit-visibility), no further action is necessary at this time.

Feel free to open a new thread if you have any further questions, 

Thank you for your contribution.
